include/benchmark/benchmark.h

```diff
@@ -153,14 +154,18 @@ void Initialize(int* argc, const char** argv);
 
 // Otherwise, run all benchmarks specified by the --benchmark_filter flag,
 // and exit after running the benchmarks.
-void RunSpecifiedBenchmarks(const BenchmarkReporter* reporter = nullptr);
+void RunSpecifiedBenchmarks(const BenchmarkReporter* reporter = NULL);
```

Comment:
can you leave the nullptr stuff alone? or does it conflict with standard library benchmarking?